database/layer.go

```diff
@@ -131,6 +131,59 @@ func InsertLayer(layer *Layer) error {
 	return nil
 }
 
+// DeleteLayer delete the specified layer and every layers that are based on that one, recursively.
```

Comment:
I do use it through https://atom.io/packages/go-plus, automatically on save but it does not wrap comments / enforce line length afaik.